src/lazy/encoder/binary/v1_0/container_writers.rs

```diff
@@ -72,108 +58,48 @@ pub struct BinaryContainerValuesWriter_1_0<'value> {
 // reallocations/copies.
 const DEFAULT_CONTAINER_BUFFER_SIZE: usize = 512;
 
-impl<'value> BinaryContainerValuesWriter_1_0<'value> {
```

Comment:
🗺️ Notice that these types had a single lifetime (`'value`); this was done to work around the compiler's HRTB limitation. Now that we're using the new traits, the writers can have two lifetimes (`'value, 'top`), meaning we don't need these anymore.